iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/normalize/AffMatchAffiliationNormalizer.java

```diff
@@ -0,0 +1,68 @@
+package eu.dnetlib.iis.wf.affmatching.normalize;
+
+import java.io.Serializable;
+
+import com.google.common.base.Preconditions;
+
+import eu.dnetlib.iis.common.string.LenientComparisonStringNormalizer;
+import eu.dnetlib.iis.common.string.StringNormalizer;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchAffiliation;
+
+/**
+ * 
+ * Normalizer of {@link AffMatchAffiliation} properties
+ * 
+ * @author Łukasz Dumiszewski
+*/
+
+public class AffMatchAffiliationNormalizer implements Serializable {
+
+    
+    private static final long serialVersionUID = 1L;
+
+    
+    private StringNormalizer organizationNameNormalizer = new LenientComparisonStringNormalizer();
+    
+    private StringNormalizer countryNameNormalizer = new LenientComparisonStringNormalizer();
+    
+    private StringNormalizer countryCodeNormalizer = new LenientComparisonStringNormalizer();
+    
+    
+    
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Normalizes properties of <code>affMatchAffiliation</code> with corresponding normalizers (see
+     * setters)<br/><br/>
+     * This method <b>changes the passed object and returns it</b>. It does NOT create a new one.
```

Comment:
Is it safe? Spark creators discouage it: http://apache-spark-user-list.1001560.n3.nabble.com/is-it-okay-to-reuse-objects-across-RDD-s-td4877.html